### PR TITLE
Fix/update admin rights restrictions

### PR DESCRIPTION
**Pull request by yurii-lubynets**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1177

### CODE REVIEW

File: src/components/organisms/TimingSpace/TimingSpace.tsx [hunk 1]
```diff
@@ -29,14 +29,9 @@ export const TimingSpace: React.FC<TimingSpaceProps> = ({
...
-    (e) => {
-      e.stopPropagation();
-
-      setShowCreateEventModal();
-    },
-    [setShowCreateEventModal]
-  );
+  const onClickCreateButton = useCallback(() => setShowCreateEventModal(), [
+    setShowCreateEventModal,
+  ]);
```
Comment (yurii-lubynets) [inline src/components/organisms/TimingSpace/TimingSpace.tsx:34]: removed `stopPropagation` as `<ButtonNG/>` is not returning  onClick `event`

**Comment by yurii-lubynets** [comment]
@goran-peoski-work 

Now,  these forms for Create/Edit Space look perfect for me. And there are no extra divs 
Screenshot 2021-09-24 at 13 36 37


Screenshot 2021-09-24 at 13 36 28

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**